src/code/z_camera.c

```diff
@@ -6240,6 +6246,8 @@ s32 Camera_Demo5(Camera* camera) {
 
     sDemo5PrevAction12Frame = camera->play->state.frames;
     Camera_ChangeSettingFlags(camera, CAM_SET_CS_C, (4 | 1));
+
+    // Continue to the general OnePoint cutscene update
     Camera_Unique9(camera);
```

Comment:
im not sure if id describe this as "continue", its more like it wants to call this in addition to the stuff it just did for this frame
overall maybe just not worth commenting on (will be especially clear when we name these camera functions with better names)

Reply:
Yeah, I just wanted to highlight that this runs two "camera-update" functions, but as you mention, maybe that'll pop out once these camera-update functions are changed to better names. I can remove